.github/workflows/ci.yaml

```diff
@@ -59,6 +59,36 @@ jobs:
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
             GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: make integration-test
+  integration-merge-test:
+    needs: test-integration
+    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged
```

Comment:
Instead of this `if`, go ahead and put this job in a new file called `postmerge.yaml` and setup the `on:` triggers to run on merges to main.

Reply:
@jeffmendoza  this is done